sdk/file_sdk/Common/file_format.cpp

```diff
@@ -0,0 +1,25 @@
+#include "file_format.h"
+
+namespace mip {
+namespace file {
+
+FileFormat::FileFormat(
+    std::shared_ptr<IStream> file,
+    const string& extension)
+  : mFile(file),
+    mExtension(extension)
+{
+}
+
+const vector<mip::Tag>& mip::file::FileFormat::GetTags()
+{
+  return mTags;

```

Comment:
what ReadTags? and what logic?  we didn't talk about ReadTags function

---
In reply to: [134435478](https://github.com/AzureAD/rms-sdk-for-cpp/pull/133#discussion_r134435478) [](ancestors = 134435478)